src/helpers/references.ts

```diff
@@ -78,3 +86,77 @@ export function splitReference(ref: string): { sheetName?: string; xc: string }
 export function getFullReference(sheetName: string | undefined, xc: string): string {
   return sheetName !== undefined ? `${getCanonicalSheetName(sheetName)}!${xc}` : xc;
 }
+
+/** Converts an RC notation to an A1 notation
+ *  An RC notation is a notation where the column and row are specified by their index
+ *  instead of by the column letter and row number. There are two types of RC notations:
+ * - Absolute RC notation: RnCm where n is the row index and m is the column index
+ * - Relative RC notation: R[n]C[m] where n is the row offset relative to a base position
+ *   and m is the column offset relative to this same base position.
+ * The relative RC notation will then need to know the initial position to use as a starting
+ * point, which is why it is passed as an optional argument.
+ * In both notation, the row and column index start at 1, ie R1C1 is A1, R1C2 is B1, R2C1 is A2,
+ * etc.
+ * As in the A1 notation, an unbounded range can be specified by omitting the row or column index :
+ * - R1: all the cells in the first row (similar to 1:1)
+ * - C1: all the cells in the first column (similar to A:A)
+ */
+export function toA1(
+  reference: string,
+  basePosition?: { col: HeaderIndex; row: HeaderIndex }
+): string {
+  const { xc, sheetName } = splitReference(reference);
+  const singleCellReference = !xc.includes(":");
+  const range = xc
+    .split(":")
+    .map((xc) => {
+      let rowPart = RcRowReference.exec(xc)?.[1] ?? "";
+      if (rowPart) {
+        const relativeRow = relativeReference.test(rowPart);
+        if (relativeRow) {
+          rowPart = rowPart.substring(1, rowPart.length - 1);
+        }
+        let rowIndex = parseInt(rowPart);
+        if (isNaN(rowIndex)) {
+          throw new InvalidReferenceError();
+        }
+        if (relativeRow) {
+          if (!basePosition) {
+            throw new InvalidReferenceError();
+          }
+          rowIndex += basePosition.row + 1;
+        }
+        if (rowIndex <= 0) {
+          throw new InvalidReferenceError();
```

Comment:
because this is a generic helper, I would return undefined or a flag value instead of raising an invalidreferenceerror here, and raise it inside the `inderect` function